swift/probe/README.md

```diff
@@ -1,15 +1,175 @@
 # Moonlight
-Probe written in [swift](https://www.swift.org/)
+A multiplatform etect probe written in [swift](https://www.swift.org/).
 
-## Getting started
+## Downloading Precompiled Binaries
```

Comment:
imo redirect this section entirely to docs.prelude.org so we only need to update one spot for future changes